pkg/nad/masterplugin.go

```diff
@@ -393,3 +393,65 @@ func (plugin *MasterIPVlanPlugin) GetMasterPluginConfig() (*MasterPlugin, error)
 
 	return plugin.masterPlugin, nil
 }
+
+// MasterBoundPlugin provides struct for MasterPlugin to create a NeNetworkAttachmentDefinition Master plugin for bonded interfaces.
+type MasterBoundPlugin struct {
+	masterPlugin *MasterPlugin
+	errorMsg     string
+}
+
+// NewMasterBoundPlugin define network-attachment-interface for bond interface.
+func NewMasterBoundPlugin(name, mode string) *MasterBoundPlugin {
+	glog.V(100).Infof(
+		"Initializing new MasterBoundPlugin structure %s", name)
+
+	builder := MasterBoundPlugin{
+		masterPlugin: &MasterPlugin{
+			CniVersion:      "0.3.1",
+			Name:            name,
+			Type:            "bond",
+			Mode:            mode,
+			FailOverMac:     1,
+			LinkInContainer: true,
+			Miimon:          "100",
+			Links:           []Link{{Name: "net1"}, {Name: "net2"}},
+			Capabilities:    &Capability{IP: true},
```

Comment:
Please proivede next values as the func parameters:

- FaileOverMac
- Miimon
- Links

Reply:
Created separate functions for each value